Make `--verbose` as an alias of `--log-mach-level debug`

qiuzhong opened the pull request:
Add a new option `--verbose` to `./wpt run` command line as an alias
of `--log-mach-level debug`.
Fix: #7846

qiuzhong committed:
Make `--verbose` as an alias of `--log-mach-level debug`

Add a new option `--verbose` to `./wpt run` command line as an alias
of `--log-mach-level debug`.
Fix: #7846

jgraham reviewed (changes requested):
This only works if we are using the mach logger. In case we aren't it will cause an error, so something else is needed.

gsnedders commented:
See also the comments earlier in  #7846

qiuzhong committed:
Add more log formatter for --verbose option

Make `--verbose` supports the raw and tbpl log formatters for
the alias of `--log-*-level` option.

qiuzhong committed:
Fix CI errors

* Fix CI errors.
* Improve test cases.

qiuzhong committed:
Correct the function name for checking verbose.

qiuzhong committed:
Rename the check_verbose function

qiuzhong committed:
Fix the condition error in check_log_verbose function

qiuzhong committed:
Update check_log_verbose name for unit tests

qiuzhong commented:
@jgraham , @gsnedders , please take a look at the new commits for issue #7846. `--verbose` option should be able to support all types of loggers now, not just for the mach logger.
